modules/ROOT/pages/download.adoc

```diff
@@ -38,7 +38,7 @@ To download and install a specific version from a terminal shell:
 
 . Set the `DFX_VERSION` environment variable to the version of the {sdk-short-name} package you want to install as a prefix to `+curl+` command.
 +
-For example, to install version 0.6.25, you would run the following command:
```

Comment:
should we update 0.6.23 [here](https://github.com/dfinity/docs/pull/365/files#diff-5b9761bd3ef159589bc5de48b1f3da118e151215327ef1b79f51a34bd91b9aefR45), too?

Reply:
good catch, eagle-eye!